client/src/components/frontend/ProjectMeta.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component, PropTypes } from 'react';
+
+export default class ProjectMeta extends Component {
```

Comment:
When we choose names, we can inadvertently create a coupling between things that shouldn't be coupled. While this component is currently used on a project page, there's no reason why it couldn't be used on another model that also has meta data. By including project in the name, we imply a coupling that doesn't need to exist. Along these lines, I'd rename it to MetaAttributes, I think.